platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/migratable/ProfileMigratable.java

```diff
@@ -42,7 +44,7 @@
    * install a feature for which the dependencies have not yet been installed. We are therefore
    * forced to attempt multiple times hoping that dependencies are properly being handled by the
    * previous pass. Failures will be reported only on the last attempt. Retries will automatically
-   * stop when all apps, features, and apps are determined to be as they were at export time.
+   * stop when all features and bundles are determined to be as they were at export time.
    */
   @SuppressWarnings("PMD.DefaultPackage" /* designed to be used by TaskList within this package */)
```

Comment:
:question: Is this retry count or attempt count?

Reply:
Sure, attempt count it is ;-)